### PR TITLE
I want to go to ANTARTICA!!!!!!!

### PR DESCRIPTION
**Pull request by Songzx07**
@AngelosZou

Answers UCL-COMP0233-24-25/RSE-Classwork#3

### CODE REVIEW

**Review by azure80 — APPROVED** [review]
Nice stufff

File: antarctica/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Look
```
Comment (dpshelio) [inline antarctica/README.md:1]: The title of this file should be called Antartica

**Review by dpshelio — CHANGES_REQUESTED** [review]
:antarctica: Very nice an cool! Please, fix the small suggestion on Antartica's readme.

**Review by dpshelio — APPROVED** [review]
Yay! :tada:

**Comment by Songzx07** [comment]
Cheers to the ultimate travel guid! Hooray!

**Pull request merged**